.github/workflows/build_micropython.yml

```diff
@@ -8,6 +8,7 @@ jobs:
   build:
     name: Build ${{ matrix.port }} port
     runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
Why continue on error?

Reply:
That gcc-arm-none-eabi action is failing sporadically again on stm32/rp2 ports. This allows the other ports' checks to keep going instead of being cancelled by the failure.

Reply:
I see.   
Is it possible to set this behavior specifically for these ports?   

Because the unix port runs many tests and if one of them fails, I think it's easier to check the bottom of the log instead of trying to search in the middle.

Reply:
My understanding is that this changes nothing about how steps (e.g. run tests) work within a job. This just affects whether one of the overall jobs (e.g. the Unix port) cancels the other jobs (the other ports) if it happens to fail.

Reply:
If this is how it works - then that's good for me.